includes/cf-pro-client/client/components/GeneralSettings/logLevel.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+    <div>
+        <div class="caldera-config-group">
+            <label for="cf-pro-form-log-level-chooser">
+                Set Log level
+            </label>
+            <div class="caldera-config-field">
+                <select
+                        id="cf-pro-form-log-level-chooser"
+                        v-model="logLevelValue"
+                >
+                    <option v-for="level in logLevels" v-bind:value="level.number">
```

Comment:
You need a key attribute `v-key="level.number"`

https://vuejs.org/v2/guide/list.html#key